packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -137,6 +145,15 @@ describeNoCompat("Container", (getTestObjectProvider) => {
 			"Client details should be set with interactive as true",
 		);
 	});
+	it("Delta manager receives pong event", async () => {
+		const container = await createConnectedContainer();
+		// let initial registration of pong event happen
+		clock.tick(60000);
```

Comment:
I'd move this line (with clock.tick()) to line 154 (inside Promise lambda) - that way, if behavior changes (and we start clock only after registering callback on deltaManager), test does not need to wait another 60 seconds or so.